src/experimentConfig.ts

```diff
@@ -40,20 +39,27 @@ const experimentDistribution = {
 const generateExperimentConfigVal = <T>(key: keyof typeof experimentDistribution, configVals: T[]): T => {
   const variantProbability = experimentDistribution[key]
 
-  if (variantProbability.length !== configVals.length - 1) {
+  if (variantProbability.length !== configVals.length) {
     console.error(`Misconfigured experimentDistribution for: '${key}'`)
   } else {
+    // Distribute the probability of each config value
+    const totalProbability = variantProbability.reduce((sum, probability) => sum + probability, 0)
```

Comment:
I don't think this value is a "probability" but instead a `denomination`, so that might be a better name because that's what it is for each probability we calculate.

Reply:
I assume you are referring only to `variantDenomination` because total denomination makes no sense